src/services/lnd/index.ts

```diff
@@ -36,6 +47,15 @@ export const LndService = (): ILightningService | LightningServiceError => {
       .find((item) => item == pubkey)
   }
 
+  const lndFromPubkey = (pubkey: Pubkey): AuthenticatedLnd | LightningServiceError => {
```

Comment:
No method in this service should return `AuthenticatedLnd` - auth is an internal detail of this service.